rules/aws_cloudtrail_rules/aws_user_login_profile_modified.py

```diff
@@ -0,0 +1,22 @@
+from panther_base_helpers import aws_rule_context, deep_get
+
+
+def rule(event):
+    if event.get("eventSource", "") == "iam.amazonaws.com":
```

Comment:
This should have a conditional on the `eventName`

Reply:
ah good catch